### ISSUE TITLE
Websocket error when ws-star.discovery.libp2p.io connection fails

### ISSUE THREAD

**Issue by olizilla** [ipfs-shipyard/peer-pad#165]
Posting this here for visibility.

![screenshot 2018-05-14 11 38 47](https://user-images.githubusercontent.com/58871/39993094-9b7b43de-576c-11e8-894b-bb83d578c157.png)

See: 
- https://github.com/ipfs-shipyard/peer-pad-core/issues/13
- #114

**Comment by lgierth** [comment]
@VictorBjelkholm is working on one

**Comment by VictorBjelkholm** [comment]
Indeed! Current testing multiaddr is `/dns4/ws-star0.lon.dwebops.pub/tcp/443/wss/p2p-websocket-star`. Once we're 100% everthing is working fine and have better metrics/monitoring in place, I'll switch the records so ws-star.discovery.libp2p.io points to the new one.

**Comment by VictorBjelkholm** [comment]
It has now been deployed to ws-star.discovery.libp2p.io. But there is not a dedicated one for peerpad, missed that point last time I read this issue. Will create a new one for just peerpad.

**Comment by pgte** [comment]
@VictorBjelkholm not sure if in flux, but I tried using `/dns4/ws-star0.lon.dwebops.pub/tcp/443/wss/p2p-websocket-star` in PeerPad and I couldn't connect.
